### PR TITLE
Final Submission

### PR DESCRIPTION
**Pull request by aboongm**
**Live Demo :** [Github-page live](https://aboongm.github.io/covid-data/)

**Video Link On Youtube :** [Youtube](https://youtu.be/LebbJD1S9y8)

## API
- Select the narrative API for COVID data

## Design
- Follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
        _Colors.
       Typography: font face, size, and weight.
        Layout: composition and space between elements._

## Interactions
- Home page
        _When the page loads, the web app shows the list of countries with a COVID figure that could be filtered out by some country name.
        Along with the country name, it displays COVID data/values per country that comes from the API.
        When the user clicks (or taps) on a country, the application navigates to the details page._
- Details page
        _In the details page, the web app retrieves data from the API to show detailed data in the selected country.
        When the user clicks on the "Back" button (<), the user navigates to the home page._

## Testing requirements
- Create unit tests for pure functions in the Redux code.
- Create integration tests for the application using the React Testing Library.

## Technical requirements
- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API is stored in the Redux store.
- Filter the data that is retrieved from the API using a Filter stateless component.
- Every page (the main page and the pages for each item)has a unique route within the SPA.
- The project is deployed on GitHub pages and accessible online (the link to the online version of the app is in README file).

## Project documentation
- a description of the project.
- a demo of the project features.
- highlight some interesting pieces of code that I am proud of.

### CODE REVIEW

**Review by prolajumokeoni — COMMENTED** [review]
Hi @aboongm, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not follow   [gitflow](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Make%20sure%20that%20you%20used%20correct%20Gitflow.)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,82 @@
...
+- Axios
+- Bootstrap
+
+## Live Demo
+
+[Github-page live](https://aboongm.github.io/covid-data/)
+
+## Video Link On Youtube
+
+[Youtube](https://youtu.be/qS4A8aZegmo)
```
Comment (AmaduKamara) [inline README.md]: Good job on this project presentation so far. There are a few additions I would like for you to make to complete the requirement for the video of this project.

- [ ] Your presentation of the project is great but you missed out on pointing out a code section of the project that you're most proud of. It's required you talk about a code section on the project you are proud of. Please redo the video with all previous explanations and add that code section. 👍

File: README.md [hunk 2]
```diff
@@ -1 +1,82 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Microverse program
+- My standup team
+- Microverse team
+- My family's support
+
```
Comment (AmaduKamara) [inline README.md]: - [ ] Kindly add acknowledgment of the original design creator and a link to that design to give credit to the designer. 👍

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
```
Comment (AmaduKamara) [inline src/App.css:20]: - [ ] Kindly follow the design guidelines as much as possible. You're free to personalize the content on the design but are asked to use exact font-face, weight, and size as per the design guidelines. For this design, the original design guidelines tell that they used [Lato font](https://fonts.google.com/specimen/Lato?query=Lato). Please import this font from the google font that is in the link above and make use of it. 👍

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @aboongm,

AlonzoLeroneThisCloseGIF (2)

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Correct setup of Github workflow
✅ No linter error
✅ Clean and organized code
✅ App works as expected
✅ README is Awesomely professional 👍 
✅ Good descriptive PR
✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by t-yanick — APPROVED** [review]
Hi @aboongm, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**